src/test/infrastructure/elasticsearch/elasticsearch-values.yaml

```diff
@@ -1,27 +1,15 @@
----
+persistence:
+  enabled: false
 
-global:
-  coordinating:
-    name: coo
+secret:
+  enabled: false
 
-master:
-  replicas: 1
-  podAnnotations:
-    "atlassian.com/business_unit": "Engineering: Enterprise DC"
-  persistence:
-    enabled: false
+createCert: false
```

Comment:
Disable ssl, or else Bitbucket will need the cert it its keystore, which is an overkill for CI.